src/lib/utils/preconditions/conditions/IPreconditionCondition.ts

```diff
@@ -25,5 +31,10 @@ export interface IPreconditionCondition {
 	 * @param command The command the message invoked.
 	 * @param entries The containers to run.
 	 */
-	parallel(message: Message, command: Command, entries: readonly IPreconditionContainer[]): PreconditionContainerReturn;
+	parallel(
+		message: Message,
+		command: Command,
+		entries: readonly IPreconditionContainer[],
+		context: PreconditionContext
```

Comment:
Isn't this added required parameter a breaking change? Please document it if so.